actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/main/resources/externalresource.properties

```diff
@@ -203,4 +203,6 @@ limit_managed_user_creation=true
 managed_user_limit=30
 adminutil_base_url = http://adminutil:4000/
 adminutil_sign_endpoint = v1/sign/payload
-enable_captcha=false
\ No newline at end of file
+enable_captcha=false
+self_declared_mandatory_fields = Diksha UUID,Status,State provided ext. ID,Channel
```

Comment:
@reshmi-nair @Hari-stackroute - until the tenant APIs are available this approach is fine. But this must be leveraged in all code above (where I have previously tagged you).

Reply:
@indrajra : will take this in next release(3.3.0)